guardian/shortcuts.py

```diff
@@ -159,7 +170,7 @@ def get_perms_for_model(cls):
     return Permission.objects.filter(content_type=ctype)
 
 def get_users_with_perms(obj, attach_perms=False, with_superusers=False,
-        with_group_users=True):
+        with_group_users=True, direct_perms_only=False):
```

Comment:
There may not have been tests that cover it, but the old behavior was described in #46, which was closed as invalid. Thus a guardian user might have felt comfortable depending on that behavior. Or  might be accidentally depending on it and be surprised on upgrade.

The defense of the old behavior in #46 was the reason I created a pull request that did not break backwards compatibility. But treating it as a bugfix results in a better long term solution.